docs/README_RECON.md

```diff
@@ -0,0 +1,316 @@
+| column_name | string    | the column that should be considered for threshold reconciliation                                           | required            | "product_discount" |
+| lower_bound | string    | the lower bound of the difference between the source value and the target value                             | optional(default=0) | -5%                | 
+| upper_bound | string    | the upper bound of the difference between the source value and the target value                             | optional(default=0) | 5%                 |            
+| type        | string    | The user must specify the column type. Supports SQLGLOT DataType.NUMERIC_TYPES and DataType.TEMPORAL_TYPES. | required            | int                |
+
+### filters
+
+```python
+@dataclass
+class Filters:
+    source: str | None = None
+    target: str | None = None
+```
+
+| field_name | data_type | description                                       | required/optional      | example_value                |
+|------------|-----------|---------------------------------------------------|------------------------|------------------------------|
+| source     | string    | the sql expression to filter the data from source | optional(default=None) | "lower(dept_name)='finance'" |
+| target     | string    | the sql expression to filter the data from target | optional(default=None) | "lower(dept_name)='finance'" |
+
+### Key Considerations:
+
+1. The column names are always converted to lowercase and considered for reconciliation.
+2. Currently, it doesn't support case insensitivity and doesn't have collation support
+3. Table Transformation internally considers the default value as the column value. It doesn't apply any default
+   transformations
+   if not provided.
+   ```eg:Transformation(column_name="address",source_name=None,target_name="trim(s_address)")```
+   For the given example,
+   the source transformation is None, so the raw value in the source is considered for reconciliation.
+4. If no user transformation is provided for a given column in the configuration by default, depending on the source
+   data
+   type, our reconciler will apply  
+   default transformation on both source and target to get the matching hash value in source and target. Please find the
+   detailed default transformations here.
+5. Always the column reference to be source column names in all the configs, except **Transformations** and **Filters**
+   as these are dialect-specific SQL expressions that are applied directly in the SQL.
+6. **Transformations** and **Filters** are always should be in their respective dialect SQL expressions, and the
+   reconciler will not apply any logic
+   on top of this.
+
+[[back to top](#remorph-reconciliation)]
+
+# Guidance for Oracle as a source
+
+## Driver
+
+1. **Download `ojdbc8.jar` from Oracle:**
```

Comment:
we have maven co-ordinates, why do they need to download?
https://mvnrepository.com/artifact/com.oracle.database.jdbc/ojdbc8

Reply:
Can you change the instructions.

Reply:
I added another point for maven library installation. Let me know if you want to completely remove the download instrcution